aquasurf/activation.py

```diff
@@ -0,0 +1,148 @@
+
+# Copyright (C) 2023 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# AQuaSurF Software in commercial settings.
+#
+# END COPYRIGHT
+"""
+Directed Acyclic Graph Activation Function
+"""
+import math
+import numpy as np
+from scipy import integrate
+from scipy.stats import norm
+import tensorflow as tf
+from tensorflow.keras.layers import Layer # pylint: disable=import-error, no-name-in-module
```

Comment:
Curious why the pylint disable was needed

Reply:
I don't know, aren't we always having problems with pylint and TF?

Reply:
If you need it, you need it, but import-error is particularly egregious to disable.